pkg/cloudprovider/provider/openstack/provider.go

```diff
@@ -231,6 +236,16 @@ func (p *provider) getConfig(provSpec clusterv1alpha1.ProviderSpec) (*Config, *p
 		return nil, nil, nil, fmt.Errorf(`failed to get the value of "InstanceReadyCheckTimeout" field, error = %v`, err)
 	}
 
+	cfg.OpenstackClientTimeout, err = p.configVarResolver.GetConfigVarDurationValueOrEnvOrDefault(rawConfig.OpenstackClientTimeout, "OS_MC_CLIENT_TIMEOUT", defaultOpenstackClientTimeout)
```

Comment:
I think we should avoid adding env variables with the `OS_` prefix.
I would always assume that these are "official" ones that are e.g. exported in my rc file.

And the `MC` is kind of superfluous.

Maybe just `OPENSTACK_CLIENT_TIMEOUT`?